### PR TITLE
Added mapping for ethnicity and sex at birth mappings

### PR DESCRIPTION
**Pull request by cukarthik**
@danqrodney  I wasn't sure about the local phi vocabularies used, but i created one called ppi-sex for "SexAtBirth" mappings.

### CODE REVIEW

File: rest-api/etl/source_to_concept_map.csv [hunk 1]
```diff
@@ -24,7 +26,7 @@ BlackSpecific_Liberian|1585618|ppi-race|Liberian|38003598|Race|1970-1-1|2099-12-
 BlackSpecific_Nigerian|1585624|ppi-race|Nigerian|38003598|Race|1970-1-1|2099-12-31|\N|1
 BlackSpecific_Somali|1585619|ppi-race|Somali|38003598|Race|1970-1-1|2099-12-31|\N|1
 BlackSpecific_SouthAfrican|1585620|ppi-race|South African|38003598|Race|1970-1-1|2099-12-31|\N|1
-Hispanic_HispanicSpecific|1586156|ppi-race|Hispanic, Latino, or Spanish (For example: Columbian, Cuban, Dominican, Mexican or Mexican American, Puerto Rican, Salvadoran, etc.) *|0|\N|\N|\N|\N|2
+Hispanic_HispanicSpecific|1586156|ppi-race|Hispanic, Latino, or Spanish (For example: Columbian, Cuban, Dominican, Mexican or Mexican American, Puerto Rican, Salvadoran, etc.) *|38003563|Ethnicity|1970-1-1|2099-12-31|\N|2
 HispanicSpecific_Mexican|1585345|ppi-race|Mexican or Mexican American|0|\N|\N|\N|\N|1
```
Comment (danqrodney) [inline rest-api/etl/source_to_concept_map.csv]: Should HispanicSpecific_Mexican and HispanicSpecific_PuertoRican also have ethnicity set?

File: rest-api/etl/source_to_concept_map.csv [hunk 2]
```diff
@@ -1,3 +1,5 @@
+SexAtBirth_Male|1585846|ppi-sex|MALE|8507|Gender|1970-1-1|2099-12-31|\N|1
+SexAtBirth_Female|1585847|ppi-sex|FEMALE|8532|Gender|1970-1-1|2099-12-31|\N|1
```
Comment (danqrodney) [inline rest-api/etl/source_to_concept_map.csv]: Do we need to worry about SexAtBirth_Intersex and SexAtBirth_SexAtBirthNoneOfThese?

File: rest-api/etl/source_to_concept_map.csv [hunk 3]
```diff
@@ -24,7 +26,7 @@ BlackSpecific_Liberian|1585618|ppi-race|Liberian|38003598|Race|1970-1-1|2099-12-
 BlackSpecific_Nigerian|1585624|ppi-race|Nigerian|38003598|Race|1970-1-1|2099-12-31|\N|1
 BlackSpecific_Somali|1585619|ppi-race|Somali|38003598|Race|1970-1-1|2099-12-31|\N|1
 BlackSpecific_SouthAfrican|1585620|ppi-race|South African|38003598|Race|1970-1-1|2099-12-31|\N|1
-Hispanic_HispanicSpecific|1586156|ppi-race|Hispanic, Latino, or Spanish (For example: Columbian, Cuban, Dominican, Mexican or Mexican American, Puerto Rican, Salvadoran, etc.) *|0|\N|\N|\N|\N|2
```
Comment (danqrodney) [inline rest-api/etl/source_to_concept_map.csv]: So it seems that these are more specific codes for hispanic race, but we are missing two top-level race codes in this file: WhatRaceEthnicity_Hispanic and WhatRaceEthnicity_RaceEthnicityNoneOfThese.

It seems like the former should map to hispanic ethnicity, and should be included in this file. I'm less sure about how to handle the second.

Comment (danqrodney) [reply rest-api/etl/source_to_concept_map.csv]: Also: I think(?) we may need to keep this row as it was before, and add additional row(s) with source_vocabulary_id set to "ppi-ethnicity" or something.

We want to be able to set race_source_concept_id to the concept IDs for these rows, while setting ethnicity_concept_id to the right ethnicity... those seem like they have to be separate rows?

File: rest-api/etl/source_to_concept_map.csv (hunk 1 above)
Comment (cukarthik) [reply rest-api/etl/source_to_concept_map.csv]: Right now we don't have a mapping for this. We'll need to introduce them. I'll make that change when we create them.

File: rest-api/etl/source_to_concept_map.csv (hunk 3 above)
Comment (cukarthik) [reply rest-api/etl/source_to_concept_map.csv]: I see. So two rows with different vocabulary ids.

**Pull request merged**